content/blog/espressif-ide-lsp-support/index.md

```diff
@@ -6,9 +6,20 @@ tags: ["News", "Espressif IDE", "LLVM", "Eclipse"]
 authors:
   - "kondal-kolipaka"
 ---
+# Table of Contents
```

Comment:
The article title in the YAML header shows up in the HTML render as the first-level heading. Suggest that you make it the second-level heading.

```suggestion
## Table of Contents
```